src/main/java/com/statisticalcalculator/StatisticalCalculatorController.java

```diff
@@ -1,18 +1,22 @@
 package com.statisticalcalculator;
 
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-@Controller 
+@Controller
+@SpringBootApplication
 public class StatisticalCalculatorController {
 	
 	@RequestMapping("/start")
 	public String start() {
 		return "start";
 	}
 	
-	@RequestMapping("/")
-	public String index(){
-		return "start";
+	public static void main(String[] args) throws Exception {

```

Comment:
Looks like two concepts in one here... the main() class should not be the same as the controller class.